### DISCUSSION TITLE
Multi User with own Mail-Address

### DISCUSSION THREAD

**Discussion by co-labor-3d** [berrystephenw/OctoText#131] [Ideas]
hi everybody
We have a small makerplace with some user.
If every user has an own OctoPrint-User-Account, would it be possible to send the e-mails only to the Mail-Address(es) of the user who started the print or to the user who has activated e.g. "send me e-mails" or is logged in, etc.?
Or do you have an idea what we can do this?
If I enter e-mail-addresses, I see, that every octoprint-User has all the same e-mail-addresses...
thanks a lot and best regards, andrea

**Comment by berrystephenw** [comment]
Hi - that would require changes to the plugin and might be a pretty big change. Each user would have to have their own settings and I'd have to load them on startup - or detect a user change. Octoprint doesn't handle multi user logins in a way that I would know this easily...
